aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,53 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 2022
+
+- **Avatar**
+  - We've added Avatar [guidance and examples](https://design-system.hpe.design/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).
```

Comment:
There's nothing in the figma file for the finished? @KennyAtHPE

Reply:
Try this [link](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3027%3A61406&t=CpM6ERbKsy0lCo2p-1) instead